stack.yaml

```diff
@@ -0,0 +1,11 @@
+resolver: lts-5.18
```

Comment:
I don't commit stack.yaml files to my projects.

Reply:
I didn't realize that was your policy. Fair enough, it probably makes sense to close this PR then.